botsquare/botsquare.ino

```diff
@@ -1,35 +1,28 @@
 #include <Wire.h>
 #include <Adafruit_MotorShield.h>
-#include "utility/Adafruit_PWMServoDriver.h"
```

Comment:
From talking with John Robert it seemed it might have been used for a servo to pan the sensor in a previous iteration,  but its not used in the code from what I can see. Probably copy paste remanence, wouldn't compile for me because I didn't have the library. Plus didn't want to ask students to add a library not needed